eq-author/src/App/QuestionnairesPage/QuestionnairesTable/index.js

```diff
@@ -13,23 +13,31 @@ import Row from "App/QuestionnairesPage/QuestionnairesTable/Row";
 
 const Table = styled.table`
   width: 100%;
-  font-size: 0.9em;
+  font-size: 1em;
   border-collapse: collapse;
   table-layout: fixed;
   text-align: left;
 `;
 
 const TH = styled.th`
-  padding: 1.5em 1em;
-  color: #8e8e8e;
+  padding: 1em;
+  color: #666;
```

Comment:
Use a colour constant for this: `${colors.darkGrey}`;

Reply:
Done.